### ISSUE TITLE
Default Currency Not Matching Default Language 

### ISSUE THREAD

**Issue by BabyOilJohnson** [futo-org/android-keyboard#766]
### Use Cases

When I changed the language to English India, still the currency shows US symbol. 

### Proposal

When changing the language it should change the currency symbol of that country (for e.g. India's ₹ not $ this symbol) 

### References

#513

**Comment by Paddy5150** [comment]
I need this too - or at least give the ability for us to set our own default. Mine's stuck on $ but I need £

**Comment by zenwav** [comment]
Seconding this request

**Comment by dvux1988** [comment]
Reed this too

**Comment by dftf-stu** [comment]
Yeah, another 🇬🇧 UK user here and I honestly thought there was an error in the FUTO keyboard.  I was certain I selected "English (United Kingdom)" when initially asked to pick the layout, yet when I tap the `?123` key for the first-layer of symbols, I get:

`1  2  3  4  5  6  7  8  9  0`
`@  #  $  _  %  &  -  +  (  )`
`=\< *  "  '  :  ;  !  ? [<X]`

I'd expect `$` to be replaced with `£` on that layer when using English (UK).

It gets a bit tedious having to either tap the `=\<` button to select it from the second-tier symbols,
or hold-down on the `$` button to select it from the button pop-up each time:

`€ ¢ £ ¥ ₱`
`     $     `

**Comment by LaurentPa** [comment]
Yes ! I want this also for French ! € instead of $.

**Comment by gouenji-shuuya** [comment]
Why does this have the solved label?

**Comment by Zvonimir-FUTO** [comment]
> Why does this have the solved label?

If you hover over it, it should show a tooltip explaining it.

Basically, has been fixed but fix has not been released yet. Probably available in nightly version but not in stable yet.

**Comment by gouenji-shuuya** [comment]
> > Why does this have the solved label?
> 
> If you hover over it, it should show a tooltip explaining it.
> 
> Basically, has been fixed but fix has not been released yet. Probably available in nightly version but not in stable yet.

Oh right, thanks and sorry for the noise!

(On mobile browser, the browser link pop up comes covering it :sigh:)

**Comment by ghost** [comment]
When Can we expect this feature?
Futo Keyboard is perfect in all cases I don't want to try other FOSS keyboards. Really need this. Everytime I have copy paste Rupee symbol because it's not available in my keyboard.

Thank you for this great keyboard.

**Comment by Zvonimir-FUTO** [comment]
In next update. You can download and tryout nightly version if you are impatient.

https://docs.keyboard.futo.org/improvements/nightly

**Comment by ghost** [comment]
I tried nightly version and it did had rupee symbol for English (India) layout. Thanks a lot.

**Comment by Zvonimir-FUTO** [comment]
Fixed in v0.1.25.
